src/runtime.js

```diff
@@ -119,6 +120,7 @@ export function installRuntimeServices(global) {
   installTimerService(global);
   installVsyncService(global);
   installXhrService(global);
+  installCachedXhrService(global);
```

Comment:
Haha just copying what I see in my terminal:
> <img width="100%" alt="screen shot 2017-03-10 at 1 28 04 pm" src="https://cloud.githubusercontent.com/assets/2363700/23813830/7caf1cfc-0595-11e7-8179-8f41c6a217e9.png">

I think something might be broken with that script